### DISCUSSION TITLE
How to find out the thread network channel is changed

### DISCUSSION THREAD

**Discussion by fuxiaoming-lumi** [openthread/openthread#8948] [Q&A] [answered]
If the parent node or the attached thread network switches the channel, how does the child device(like sleepy end device) perceive the channel switch and switch to the new channel automatically .

**Comment by jwhui** [comment, marked as answer]
See [Thread 1.3.0](https://www.threadgroup.org/support#specifications) Section 8.7.7 "Orphaned End Devices".

In short, orphaned end devices (a child that cannot find a neighboring router to attach to) sends MLE Announce messages on different channels. A neighboring router that receives this MLE Announce message will respond to the child indicating the channel to communicate on.
